tests/acceptance/features/bootstrap/OccContext.php

```diff
@@ -364,6 +369,16 @@ public function deleteLastBackgroundJobUsingTheOccCommand($job) {
 		$this->lastDeletedJobId = $match;
 	}
 
+	/**
+	 * List created local storage mount
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function listLocalStorageMount() {
+		\OccContext::invokingTheCommand('files_external:list --output=json');
```

Comment:
```suggestion
		$this->invokingTheCommand('files_external:list --output=json');
```
when you are inside occ context its better to use `this`